### PR TITLE
feat: migration banner

### PR DESCRIPTION
**Pull request by alfonsobries**
## Summary

PR. https://app.clickup.com/t/39xf25f

## Checklist

-   [ ] My changes look good in both light AND dark mode
-   [ ] The change is not hardcoded to a single network, but has multi-asset in mind
-   [ ] I checked my changes for obvious issues, debug statements and commented code
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

File: src/domains/dashboard/components/Migration/MigrationBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+							onClick={learnMoreClickHandler}
+						>
+							{t("COMMON.LEARN_MORE")}
+						</Button>
+					</div>
+				</div>
+
+				<div className="hidden w-[304px] flex-shrink-0 pt-2 pb-4 md:block lg:w-[475px]">
+					<PolygonMigrationBannerLight className="block w-full dark:hidden" />
+					<PolygonMigrationBannerDark className="hidden w-full dark:block" />
```
Comment (dated) [inline src/domains/dashboard/components/Migration/MigrationBanner.tsx:45]:
```suggestion
					<Image name="PolygonMigrationBanner" useAccentColor={false} />
```

File: src/domains/dashboard/components/Migration/MigrationBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+export const MigrationBanner = () => {
+	const { t } = useTranslation();
+
+	return (
+		<div
+			data-testid="MigrationBanner"
+			className="mb-4 bg-theme-primary-100 text-theme-secondary-700 dark:border-theme-secondary-800 dark:bg-theme-secondary-900 dark:text-theme-secondary-500 sm:mb-0 sm:dark:bg-black"
+		>
+			<div className="flex items-center px-8 md:px-10 lg:container lg:mx-auto">
+				<div className="max-w-2xl flex-1  py-6">
```
Comment (dated) [inline src/domains/dashboard/components/Migration/MigrationBanner.tsx:20]:
```suggestion
				<div className="max-w-2xl flex-1 py-6">
```

File: src/domains/dashboard/components/Migration/MigrationBanner.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { Button } from "@/app/components/Button";
+import { Link } from "@/app/components/Link";
+import { images } from "@/app/assets/images";
+const { PolygonMigrationBannerDark, PolygonMigrationBannerLight } = images.common;
+const learnMoreClickHandler = () => {
+	const link = "https://ardenthq.com/blog";
+	window.open(link);
+};
```
Comment (dated) [inline src/domains/dashboard/components/Migration/MigrationBanner.tsx:10]: Can be based on the `useLink()` hook, e.g.

```ts
const { openExternal } = useLink();
```

**Review by dated — CHANGES_REQUESTED** [review]

**Comment by alfonsobries** [comment]
@dated ✅

**Comment by dated** [comment]
> @dated ✅

did you forget to push? :trollface:

**Comment by alfonsobries** [comment]
@dated apparently yes (and also lost the haha) but just updated again

**Review by ItsANameToo — APPROVED** [review]

**Review by samharperpittam — CHANGES_REQUESTED** [review]
Header should be 18px on mobile

Design
image

Imp
image

**Review by samharperpittam — CHANGES_REQUESTED** [review]
The text within the banner should be semi-bold

image

**Review by dated — APPROVED** [review]

**Pull request merged**